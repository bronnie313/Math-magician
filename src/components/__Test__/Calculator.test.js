import React from 'react';
import { render, screen } from '@testing-library/react';
import Cal from '../Calculator';

describe('Cal component', () => {
  it('renders the calculator buttons', () => {
    render(<Cal />);
    const calculatorButtons = screen.getAllByRole('button');
    expect(calculatorButtons.length).toBeGreaterThan(0);
  });
});
