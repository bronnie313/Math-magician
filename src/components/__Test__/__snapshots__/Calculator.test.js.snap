// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component renders correctly 1`] = `
<div
  className="info"
>
  <div>
    <h2
      className="cal"
    >
      Let's do some Math
    </h2>
  </div>
  <section>
    <div
      className="screen"
    >
      <h2>
        0
      </h2>
    </div>
    <article
      className="keys"
    >
      <input
        onClick={[Function]}
        type="button"
        value="AC"
      />
      <input
        onClick={[Function]}
        type="button"
        value="+/-"
      />
      <input
        onClick={[Function]}
        type="button"
        value="%"
      />
      <input
        className="lastColumn"
        onClick={[Function]}
        type="button"
        value="รท"
      />
      <input
        onClick={[Function]}
        type="button"
        value="7"
      />
      <input
        onClick={[Function]}
        type="button"
        value="8"
      />
      <input
        onClick={[Function]}
        type="button"
        value="9"
      />
      <input
        className="lastColumn"
        onClick={[Function]}
        type="button"
        value="x"
      />
      <input
        onClick={[Function]}
        type="button"
        value="4"
      />
      <input
        onClick={[Function]}
        type="button"
        value="5"
      />
      <input
        onClick={[Function]}
        type="button"
        value="6"
      />
      <input
        className="lastColumn"
        onClick={[Function]}
        type="button"
        value="-"
      />
      <input
        onClick={[Function]}
        type="button"
        value="1"
      />
      <input
        onClick={[Function]}
        type="button"
        value="2"
      />
      <input
        onClick={[Function]}
        type="button"
        value="3"
      />
      <input
        className="lastColumn"
        onClick={[Function]}
        type="button"
        value="+"
      />
      <input
        className="zero"
        onClick={[Function]}
        type="button"
        value="0"
      />
      <input
        onClick={[Function]}
        type="button"
        value="."
      />
      <input
        className="lastColumn"
        onClick={[Function]}
        type="button"
        value="="
      />
    </article>
  </section>
</div>
`;
